### ISSUE TITLE
Profit amount % of sales amount?

### ISSUE THREAD

**Issue by data2base** [opensourcepos/opensourcepos#2599]
IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

 Reporting an issue of an unmodified OSPOS installation
 Checked open and closed issues and no similar issue was already reported (please make sure you searched!)
 Read README, WHATS_NEW, INSTALL.md and UPGRADE
 Read the FAQ for any known install and/or upgrade gotchas (in specific PHP extensions installed)
 Read the wiki
 Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
 Aware the installation code is in bintray (see README), and GitHub master is for developers only and therefore not complete nor stable
Installation information
.Browser: Chrome
.Server Software: Apache
.PHP Version: 7.0.33
.DB Version: 5.7.28-log
.Server Port: 443
.OS: Linux depro4.fcomet.com 2.6.32-954.3.5.lve1.4.71.el6.x86_64 #1 SMP Fri Sep 6 06:22:16 EDT 2019 x86_64

OSPOS version is: 3.3.0 - 0b9a76
OSPOS git commit hash is:
PHP version is:
MySQL or MariaDB version is:
OS and version is:
WebServer is:
Selected language is:
(If applicable) Docker installation:
(If applicable) Installation package for the LAMP/LEMP stack is:

Question

Hi!

I want to use the system for a market/ expo. 
So basically profit for a market/ expo works on a % of the total sale amount.

It this possible?

We can even go to link a % profit to a specific supplier?

Please advise.
